src/subscript/csv_stack/csv_stack.py

```diff
@@ -65,133 +145,198 @@ def get_parser():
     parser.add_argument(
         "--keepminimal",
         action="store_true",
-        help="Keep only REAL, ENSEMBLE, DATE and unpivoted columns",
+        help=(
+            "Keep only REAL, ENSEMBLE, DATE and unpivoted columns. "
+            "Implies dropping constant columns"
+        ),
```

Comment:
this list doesn't ceover the whole `ALWAYS_KEEP` list. Of course, these are the most likely ones from export with e.g. `CSV_EXPORT` / f`mu-ensemble`, but a bit misleading

Reply:
Fair enough. Removing the legacy column names from `ALWAYS_KEEP`. Implies an API break, but anyone relying on this would be a big surprise.